src/styles/components/_stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars-list {
+  display: flex;
+}
+
+.stars-item {
+  width: 16px;
+  height: 16px;
+  background: url('../images/star.svg') center no-repeat;
+}
+
+.stars-list--active .stars-item:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card-stars_raiting {
```

Comment:
There is a typo in the class name `.card-stars_raiting`. It should be `.card-stars_rating` to correctly spell 'rating'.